01-lets_get_started/01.rb

```diff
@@ -0,0 +1,28 @@
+# To make a empty square
+class Square
+  def initialize(side)
+    @side = side
+    @count = side - 2
```

Comment:
Think: Does `@count` **mean** anything to the problem domain? Shouldn't that variable be called something like `@cavity_width` or `@gap_length` or something similar, so that the lines below will read more fluidly? For instance:

`'#' + ' ' * @count + '#'`

OR

`'#' + ' ' * @cavity_width + '#'`